recipes-images/packagegroups/packagegroup-hostmobility-net-minimal.bb

```diff
@@ -7,6 +7,5 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 
 RDEPENDS:${PN} = " \
     ppp \
-    ntpdate \
```

Comment:
why is ntpdate removed?

Reply:
This is the motivation:

"ntpdate not found in Mickledore layers from Variscite."

Reply:
if this layer should be compatible with kirkstone it must be included. what i can see it is still possible to use.https://git.openembedded.org/meta-openembedded/tree/meta-networking/recipes-support/ntp?h=mickledore